BridgeAppSDK/SBAActivityTableViewController.swift

```diff
@@ -156,6 +167,28 @@ public class SBAActivityTableViewController: UITableViewController, SBASharedInf
     
     // MARK: Protected subclass methods
     
+    public func shouldShowTaskForSchedule(schedule: SBBScheduledActivity) -> Bool {
+        return true
```

Comment:
Is this a stub to be filled in later, or a base class implementation to be overridden?